pkgs/applications/video/obs-studio/wlrobs.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchhg, obs-studio, wayland }:
+
+stdenv.mkDerivation rec {
+  pname = "wlrobs";
+  version = "82e2b93c6f662dfd9d69f7826c0096bef585c3ae";
```

Comment:
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
Sorry, I've fixed this I believe.